README.md

```diff
@@ -19,99 +19,17 @@ The following diagram simplifies how FortiSOAR's Continuous Delivery works:
 
 ![Continuous Delivery Flowchart](./docs/res/ci-cd-flow.svg)
 
-## Intended Audience
+## Who Should Read This
 
-This document is meant for content developers and FortiSOAR administrators who build FortiSOAR content to suit their daily requirements. Customizations to FortiSOAR content can be pushed to source control in the development environment. After ironing out any issues with customizations and when confident about the custom content, the same can be pushed to the production source control to be delivered/deployed on the production FortiSOAR instance.
```

Comment:
Customizations to FortiSOAR content can be pushed to source control in the development environment. => Customizations to FortiSOAR content can be pushed to the development source control branch.

Reply:
Content removed